ci-operator/step-registry/upi/conf/vsphere/ova/windows/upi-conf-vsphere-ova-windows-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+# TODO: read VM template information from environment variable
+VM_TEMPLATE="windows-golden-images/windows-server-2004-template"
```

Comment:
windows-server-2022-template was recently deployed, do we need to update to refer to that here?

Reply:
[Yes](https://github.com/openshift/windows-machine-config-operator/pull/892#discussion_r799491826), once https://github.com/openshift/windows-machine-config-operator/pull/892 merges.